src/main/java/seedu/address/logic/commands/FindMedConCommand.java

```diff
@@ -17,9 +17,8 @@ public class FindMedConCommand extends Command {
     public static final String COMMAND_WORD_INSENSITIVE = "findmedcon";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all patients with medical conditions that "
-            + "contains the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: "
-            + "KEYWORD [MORE KEYWORDS]\n"
+            + "contains the specified keywords (case-insensitive) and displays them.\n"
+            + "Parameters: KEYWORD [MORE KEYWORDS]...\n"
```

Comment:
Actually adding the params on a new line is more inline with the other command usage format messages